muduo/net/Channel.h

```diff
@@ -49,6 +51,9 @@ class Channel : noncopyable
   void setErrorCallback(EventCallback cb)
   { errorCallback_ = std::move(cb); }
 
+  void setEventsCallback(const EventsCallback& cb)
```

Comment:
Follow the convention here: pass-by-value and std::move(), also no blank line before this function.